frontend/package.json

```diff
@@ -14,6 +14,9 @@
     "react-scripts": "3.4.1",
     "uswds": "^2.8.1"
   },
+  "engines": {
+    "node": "^12"
```

Comment:
It's been a long time since I've looked at caret ranges. This will allow both patches and minor updates (>=12.0.0 <13.0.0), right?

It seems likely to me that we'll want the docker node version to match the version specified in package.json. Unfortunately though, I'm not seeing an image tag for circleci/node the species only the major release. They do appear to have image tags that have . though. So we could do circleci/node:12.18 as the image and "node": "^12.18" here and still get the new patches.

Reply:
> It's been a long time since I've looked at caret ranges. This will allow both patches and minor updates (>=12.0.0 <13.0.0), right?

Yep!

> It seems likely to me that we'll want the docker node version to match the version specified in package.json. Unfortunately though, I'm not seeing an image tag for circleci/node that specifies only the major release. They do appear to have image tags that have the minor release though. So we could do circleci/node:12.18 as the image and "node": "^12.18" here and still get the new patches.

Makes sense to me to use "^12.18"